A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -844,6 +848,40 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         SimVar.SetSimVarValue("L:A320_Neo_ADIRS_ALIGN_LIGHT_3", "Bool", (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_3", "Enum") == 1 && ADIRSState != 2) );
 
     }
+
+    SetBalanceData(data) {
+        this._balanceData = data.balance;
+    }
+
+    trySetZFWCG(input, useLbs = false) {
```

Comment:
instead of copying the function and adding your mods, we've been copying the whole file over into the mod, and making our changes to the original function. please do that unless you have a reason why not to

Reply:
FMCMainDisplay.js

Reply:
but because its a shared file, it needs special care, see how to copied over `BaseNDCompass.js`